compiler/modules/nback/scaffold.bal

```diff
@@ -115,6 +115,7 @@ class Scaffold {
     private final llvm:FunctionDefn llFunc;
     private final DISubprogram? diFunc;
     private DILocation? noLineLocation = ();
+    private final DebugLevel debugLevel;
```

Comment:
This field should be in ModuleDi and should be boolean debugFull. debugLevel 0 is indicated by have a null ModuleDI.

Reply:
fixed